projects/packages/jetpack-mu-wpcom/babel.config.json

```diff
@@ -0,0 +1,11 @@
+{
+	"plugins": [
+		[
+			"@babel/plugin-transform-react-jsx",
```

Comment:
Converts JSX to react function calls.

<div>Example<div> -> React.createElement("div", null, "Example");